src/trans/gpu/internal/ftinv_mod.F90

```diff
@@ -77,7 +92,12 @@ SUBROUTINE FTINV(ALLOCATOR,HFTINV,PREEL_COMPLEX,PREEL_REAL,KFIELD)
 
     INTEGER(KIND=JPIM) :: KGL
 
+#ifdef IN_PLACE_FFT
     PREEL_REAL => PREEL_COMPLEX
+#else
+    CALL ASSIGN_PTR(PREEL_REAL, GET_ALLOCATION(ALLOCATOR, HFTINV%HREEL_REAL),&
+      & 1_C_SIZE_T, INT(KFIELD*D%NLENGTF*SIZEOF(PREEL_REAL(1)),KIND=C_SIZE_T))
```

Comment:
and here..
Perhaps there are more instances in the code. This seems like a typical copy-paste-edit line.